packages/inertia-vue/src/app.js

```diff
@@ -15,6 +15,10 @@ export default {
       type: Function,
       required: true,
     },
+    resolveErrors: {
+      type: Function,
+      required: false,
```

Comment:
So, I wonder if we should set the default here (instead of in Inertia core), simply because that's what we did with `transformProps`.

```suggestion
      default: page => (page.props.errors || {}),
```